ros/bbn_integration_py/setup.py

```diff
@@ -0,0 +1,29 @@
+from setuptools import setup, find_packages
+
+package_name = 'bbn_integration_py'
+
+setup(
+    name=package_name,
+    version='0.1.0',
+    packages=find_packages(include=(package_name+"*",)),
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    maintainer='Paul Tunison',
+    maintainer_email='paul.tunison@kitware.com',
+    # ---
+    # Shared with properties in package.xml.
+    description='ROS2 nodes in python for BBN integration support',
+    license='BSD 3-Clause',
+    # ---
+    tests_require=['pytest'],
```

Comment:
When do these pytest scripts run? Are they run every time the package is built or just manually?

Reply:
These are default values as populated by the `ros2 pkg create` tooling when specifying that it's going to be an ament_python package. Our other packages like `ros/angel_system_nodes` have this set here as well. This would be used in CI testing if that was set up (oh boy do we have issues to fix in that case).